environments/production/inventory.ini

```diff
@@ -165,12 +169,34 @@ hqcelery3.internal-va.commcarehq.org
 [hqcelery4]
 hqcelery4.internal-va.commcarehq.org
 
+
+[celery0]
+10.202.12.202 hostname=celery0-production ec2=yes
+
+[celery1]
+10.202.12.214 hostname=celery1-production ec2=yes #datadog_integrations=celery
```

Comment:
can we keep `datadog_integrations=celery` turned on here?

Reply:
I was planning on turning it on once we shut off the other ones. Rackspace hqcelery1 still has this integration, and I didn't know what would happen if we turned it on for both

Reply:
Oh, I was thinking this was `datadog_integration_tcp_check`, which is different. Never mind.